adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleBanner.m

```diff
@@ -101,154 +98,92 @@ - (void)requestBannerAd {
-    NSError *error = GADMAdapterVungleErrorWithCodeAndDescription(
-        GADMAdapterVungleErrorInvalidServerParameters,
-        @"Missing or invalid Placement ID configured for this ad source instance in the AdMob or "
-        @"Ad Manager UI.");
+    NSError *error = GADMAdapterVungleInvalidPlacementErrorWithCodeAndDescription();
     _adLoadCompletionHandler(nil, error);
     return;
   }
 
-  if (![[GADMAdapterVungleBiddingRouter sharedInstance] isSDKInitialized]) {
+  if (![VungleAds isInitialized]) {
     NSString *appID = [GADMAdapterVungleUtils findAppID:_adConfiguration.credentials.settings];
-    [[GADMAdapterVungleBiddingRouter sharedInstance] initWithAppId:appID delegate:self];
+    [[GADMAdapterVungleRouter sharedInstance] initWithAppId:appID delegate:self];
     return;
   }
 
   [self loadAd];
 }
 
 - (void)loadAd {
-  NSError *error = [[GADMAdapterVungleBiddingRouter sharedInstance] loadAdWithDelegate:self];
-  if (error) {
-    _adLoadCompletionHandler(nil, error);
-  }
-}
-
-- (NSError *)renderAd {
-  VungleAdNetworkExtras *extras = (VungleAdNetworkExtras *)[_adConfiguration extras];
-  NSMutableDictionary *options = nil;
-  if (extras) {
-    options = [[NSMutableDictionary alloc] init];
-    if (extras.muteIsSet) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(options, VunglePlayAdOptionKeyStartMuted,
-                                                        @(extras.muted));
-    }
-    if (extras.userId) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(options, VunglePlayAdOptionKeyUser,
-                                                        extras.userId);
-    }
-    if (extras.flexViewAutoDismissSeconds) {
-      GADMAdapterVungleMutableDictionarySetObjectForKey(
-          options, VunglePlayAdOptionKeyFlexViewAutoDismissSeconds,
-          @(extras.flexViewAutoDismissSeconds));
-    }
-  }
-  NSError *bannerError = nil;
-  [VungleSDK.sharedSDK addAdViewToView:_bannerView
-                           withOptions:options
-                           placementID:self.desiredPlacement
-                              adMarkup:[self bidResponse]
-                                 error:&bannerError];
-  return bannerError;
-}
-
-- (void)cleanUp {
-  if (_didBannerFinishPresenting) {
-    return;
-  }
-  _didBannerFinishPresenting = YES;
-
-  [VungleSDK.sharedSDK finishDisplayingAd:self.desiredPlacement adMarkup:self.bidResponse];
-  [[GADMAdapterVungleBiddingRouter sharedInstance] removeDelegate:self];
-  _bannerView = nil;
-}
-
-#pragma mark - GADMAdapterVungleDelegate delegates
-
-- (NSString *)bidResponse {
-  return [_adConfiguration bidResponse];
-}
-
-- (GADAdSize)bannerAdSize {
-  return _bannerSize;
+  _bannerAd = [[VungleBanner alloc]
+      initWithPlacementId:self.desiredPlacement
+                     size:GADMAdapterVungleConvertGADAdSizeToBannerSize(_bannerSize)];
+  _bannerAd.delegate = self;
+  VungleAdsExtras *extras = [[VungleAdsExtras alloc] init];
```

Comment:
We're having build issues where `VungleAdsExtras` cannot be found:

```
Use of undeclared identifier 'VungleAdsExtras'
```

Reply:
This was support for the watermark. I thought that it was going to be released soon on our end so I included this back in. Let me roll this commit back.

Reply:
@stanley-vungle Just to confirm - so watermark for Vungle bidding ads isn't supported yet, correct? Would you know when will it be supported?